services/app-api/src/resolvers/submitHealthPlanPackage.test.ts

```diff
@@ -225,7 +225,7 @@ describe('submitHealthPlanPackage', () => {
 
         expect(submitResult.errors).toBeUndefined()
         expect(mockEmailer.sendEmail).toHaveBeenCalledTimes(2)
-    })
+    }, 20000)
```

Comment:
were you seeing timeout failures here? I’m a little surprised, shouldn’t this all be real fast?